lib/manageiq/cross_repo/runner.rb

```diff
@@ -3,9 +3,9 @@
 
 module ManageIQ::CrossRepo
   class Runner
-    attr_reader :test_repo, :core_repo, :gem_repos, :script_cmd
+    attr_reader :test_repo, :core_repo, :gem_repos, :test_suite, :script_cmd, :install_cmd
 
-    def initialize(test_repo, repos, script_cmd = "")
+    def initialize(test_repo, repos, test_suite = "", script_cmd = "")
```

Comment:
FWIW, I've found keyword args a LOT more convenient with respect to CLI args. So...
```suggestion
    def initialize(test_repo:, repos:, test_suite: "", script_cmd: "", **_)
```

Then up above

```diff
-  ManageIQ::CrossRepo.run(opts[:test_repo], opts[:repos], opts[:test_suite], opts[:script_cmd])
+ ManageIQ::CrossRepo.run(opts)
```